index.js

```diff
@@ -60,16 +60,21 @@ const next = () => {
   const nextBoard = board
   board.forEach(row => {
     row.forEach(cell => {
-      // if the
-      if (board[row][cell]) {
-
+      const neighbors = liveNeighbors(row, cell)
+      // if the spot is truthy, and has 2 or 3 neighbors
+      if (board[row][cell] && (neighbors == 2 || neighbors == 3)) {
+      nextBoard[row][cell] = board[row][cell]
+      // if the spot is dead but has exactly 3 neighbors, it next becomes alive
+      } else if (board[row][cell] === 0 && neighbors === 3) {
```

Comment:
Disclaimer: This is just a stylistic preference, but any time you are checking to see if the value of something is === 0, you can write it as:  `if(!board[row][cell] && ...rest of code)` since 0 evaluates falsey.

Reply:
Oh yeah that's why I chose `1` and `0` to begin with BUT theres also the chance that those values would be `undefined` which is also "falsey" - so I only wanted that if it was specifically zero